include/ONV.hpp

```diff
@@ -3,25 +3,36 @@
 
 
 #include <Eigen/Dense>
+
 #include "common.hpp"
 
 
 
 namespace GQCG {
 
 
+/**
+ *      An ONV in quantum chemistry is a string of creation operators acting on top of a vacuum state.
+ *      An example for 3 alpha electrons in a Fock space spanned by 4 spatial orbitals is
+ *          a_1^\dagger a_2^\dagger a_3^\dagger |vac> = |1,1,1,0>
+ *
+ *      In this code, we are using REVERSE LEXICAL notation, i.e. bitstrings are read from right to left. This means that the
+ *      least significant bit relates to the first orbital. Using this notation is how normally bits are read, leading
+ *      to more efficient code. As is also usual, the least significant bit has index 0.
+ *          The previous example is then represented by the bit string "0111" (7).
+ */
 class ONV {
 private:
     const size_t K;  // number of spatial orbitals
     const size_t N;  // number of electrons
     size_t unsigned_representation;  // unsigned representation
-    VectorXs occupation_indexes;  // the occupied orbital electron indexes
+    VectorXs occupation_indices;  // the occupied orbital electron indexes (of length N)
```

Comment:
maybe add some more information about the occupation indices: it is an array of N elements in which array[j] gives the occupied orbital index for electron j